File: src/test/resources/testData/testprojects/intellij-integration/src/java/org/pantsbuild/testproject/failures/simple/HelloWorld.java
// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
// Licensed under the Apache License, Version 2.0 (see LICENSE).

import java.lang.Deprecated;

/**
 * ./pants compile testData/testprojects/intellij-integration/src/java/org/pantsbuild/testproject/failures/simple:simple
 */
public class HelloWorld {
  @Deprecated
  public static deprecatedMethod() {
    /* bad code to generate warnings and errors */
    int unusedVar;
    if (1 == 2) {
    /* missed bracket: } */
  }
  public static void main(String[] args) {
    System.out.println("Hello World!") /* missed semicolon: ; */
  }
}

File: src/test/resources/testData/testprojects/intellij-integration/src/scala/org/pantsbuild/testproject/failures/simple/HelloWorld.scala
// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
// Licensed under the Apache License, Version 2.0 (see LICENSE).

package org.pantsbuild.testproject.failures.simple;

/**
 * ./pants compile.scala testData/testprojects/intellij-integration/src/scala/org/pantsbuild/testproject/failures/exe:exe
 */

object HelloWorld {
  /* bad code to generate errors */
  def badMethod() {
    if () /* missed brackets: {} */
  }

  def main(args: Array[String]) {
      println("Hello, world!")
    /* missed bracket: } */
}
